radsat-sk/operation/utility/crc.c

```diff
@@ -0,0 +1,257 @@
+/**********************************************************************

```

Comment:
can you send me a link to where you got this code and any additional documentation so I can update the design document?

Reply:
https://barrgroup.com/downloads/code-crc-c